docs/user/guides/best-practice/index.rst

```diff
@@ -0,0 +1,36 @@
+How-to guides: best practice
+============================
+
+Over the years,
+we have become familiar with a number of methods that work well and which we consider **best practice**.
```

Comment:
```suggestion
These guides share best practices that we've learned for managing documentation.
If you need ideas for how to get started in various areas of documentation,
this is a great place.
```

Reply:
Can you update the suggestion? I can't quite make out what the intention was with the change.

Reply:
Not hugely important, just trying to clarify what the content is a bit more. But it's not my best writing, for sure.